HeroArena-App/src/app/roster-display/roster-display.component.ts

```diff
@@ -17,15 +19,37 @@ export class RosterDisplayComponent implements OnInit {
 
   constructor(private rs: RosterService, private as: AuthenticateService) { }
 
+  selectedGladiator = '';
+  // gladiators: Array<Gladiator> = [];
+  gladiators: any = [];
   ngOnInit() {
-    //this.rs.getGladiators();
     this.rs.getUserGladiators(this.as.getUser().id).subscribe((userObj: Object) => {
-      console.log(userObj);
+      // console.log(userObj);
       this.rs.setGladiators(userObj);
+
+      this.gladiators = userObj;
+
+      // console.log(userObj[0]);
```

Comment:
Let's try not to leave code comments, or at least write a comment about the significance of the code comments.